src/Element/NoteReaction.tsx

```diff
@@ -53,8 +57,9 @@ export default function NoteReaction(props: NoteReactionProps) {
         showHeader: ev?.Kind === EventKind.Repost,
         showFooter: false,
     };
+    const isOpMuted = root && isMuted(root.pubkey)
```

Comment:
the `p` tag check and `root.pubkey` should be the same, do you think its needed to have both checks?

Reply:
No, my bad